packages/pigeon/example/app/ios/Runner/AppDelegate.swift

```diff
@@ -5,11 +5,47 @@
 import Flutter
 import UIKit
 
+// #docregion swift-class
+// This extension of Error is required to do use FlutterError in any swift code.
```

Comment:
Swift

(I noticed this just as I was submitting the review. Maybe sweep the PR diffs for references to language names in English text, and make sure they are always capitalized.)